--- qgis-3.42.0/src/core/CMakeLists.txt.orig	2025-03-06 20:03:17.945633451 +0000
+++ qgis-3.42.0/src/core/CMakeLists.txt	2025-03-06 20:03:23.840552489 +0000
@@ -2193,7 +2193,6 @@
   )
 else()
   if ("${CMAKE_CXX_COMPILER_ID}" MATCHES "Clang" AND (CMAKE_BUILD_TYPE MATCHES Release OR CMAKE_BUILD_TYPE MATCHES RelWithDebInfo))
-    set_source_files_properties(${CMAKE_CURRENT_BINARY_DIR}/qgsexpression_texts.cpp PROPERTIES COMPILE_FLAGS "-O1")
     set_source_files_properties(proj/qgscoordinatereferencesystem.cpp PROPERTIES COMPILE_FLAGS "-mllvm -inline-threshold=128")
   endif()
 endif()
